### ISSUE TITLE
Disable "Delete all content" when deleting users

### ISSUE THREAD

**Issue by blogtutor** [blogtutor/blog-tutor-support#276]
If there is any content associated with a user, after the user is selected for deletion, WordPress will show a page like this:

![image](https://github.com/blogtutor/blog-tutor-support/assets/24479276/a83fda45-b10b-4e93-b895-f9eed5b8a7b5)

Our clients often don't realize that "content" means more than `Posts` -- including any uploaded media, or even things like settings in the Customizer.  As a result, deleting a user and their content can break a site very badly. 

Until we have time to submit a proper patch to core, let's completely hide the "Delete all content" option with CSS.

**Comment by blogtutor** [comment]
Hmm, looks also like there might be two hooks in users.php that we could leverage if we really want to improve this:

https://developer.wordpress.org/reference/files/wp-admin/users.php/

--

[users_have_additional_content](https://developer.wordpress.org/reference/hooks/users_have_additional_content/)
Filter Hook: Filters whether the users being deleted have additional content associated with them outside of the `post_author` and `link_owner` relationships.

Source: wp-admin/users.php:311

--

[delete_user_form](https://developer.wordpress.org/reference/hooks/delete_user_form/)
Action Hook: Fires at the end of the delete users form prior to the confirm button.

Source: wp-admin/users.php:436

--

**Comment by blogtutor** [comment]
This should do it -- we just need to add this on the correct dashboard page. 

```
#delete_option0, label[for=delete_option0] {
 display: none
}
```

**Comment by blogtutor** [comment]
Closed with Release 2.1.
